src/explorer.ts

```diff
@@ -453,7 +469,7 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
                     name: 'build configs'
                 },
             } as OpenShiftObject;
-            result.push(pods,
+            result.push(pods, routes,
```

Comment:
The `Routes` App. Explorer tree item is an Openshift specific, so it should be moved inder `if (isOpenshiftCluster) {...}` here after.